api/API.md

```diff
@@ -39,29 +39,34 @@ The root endpoint for all API requests for version 1.0 of the API is `/api/v1`
 
 ### RecrudescenceTest
 
-Given an input `.csv` file with [drug testing data in the proper format (TODO)](), return estimates of which patients specified in the file are reinfections or recrudescences.
+Given an input `.xlsx` file with [drug testing data in the proper format (TODO)](), return estimates of which patients specified in the file are reinfections or recrudescences.
 
-*Method(s):* `PUSH`
+*Method(s):* `POST`
 
 *URL Endpoint:* `/api/v1/recrudescences`
 
-#### PUSH
+#### POST
 
 **Input Parameters (URL):**
 
 -   (optional) `iterations` (integer): The number of iterations to run the algorithm for on the data; higher values are more accurate but take longer to complete. Defaults to 10000.
 
 **Input Parameters (Data):**
 
--   `file` (.csv file, FormData): The file containing the testing data to be processed.
+-   `file` (.xlsx file, FormData): The file containing the testing data to be processed.
+
+**Input Parameters (URL):**
```

Comment:
Don't create a 2nd "Input Parameters (URL)" section; instead, add this as a bullet point under the 1st "Input Parameters (URL)"

Reply:
Makes sense, change made